thirdeye/thirdeye-pinot/src/main/resources/migrations.xml

```diff
@@ -334,4 +334,33 @@
     </addColumn>
   </changeSet>
 
+  <changeSet id="12" author="pjaiswal">
+    <addColumn tableName="anomaly_results">
+      <column name="anomaly_feedback_id" type="bigint" defaultValue="null">
+        <constraints nullable="true"/>
+      </column>
+    </addColumn>
+
+    <dropColumn tableName="anomaly_results" columnName="feedback"/>
+
+    <createTable tableName="anomaly_feedback">
+      <column name="id" type="bigint" autoIncrement="true">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="feedback" type="varchar(100)"/>
```

Comment:
rename to feedback_type so it's clearer that this is different from comments/description/status.